cpp/src/slice2cs/CsUtil.cpp

```diff
@@ -165,12 +165,12 @@ Slice::CsGenerator::fixId(const string& name, unsigned int baseTypes, bool mangl
     }
     vector<string> ids = splitScopedName(name);
     vector<string> newIds;
-    for (vector<string>::const_iterator i = ids.begin(); i != ids.end(); ++i)
+    for (auto& id : ids)
```

Comment:
Should this be:
```suggestion
    for (const auto& id : ids)
```
? I feel like there's alot of places where the `const` is missing, but wanted to ask if there was a reason, or if this was just a shortcoming of the tool before I point them out.

Reply:
Yes, it should be const. Will fix.

Reply:
Fixed.